aspnetcore/fundamentals/w3c-logger/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: W3C Logger in .NET Core and ASP.NET Core
+author: wtgodbe
+description: Learn how to create server logs in the W3C standard format.
+monikerRange: '>= aspnetcore-6.0'
+ms.author: wigodbe
+ms.custom: mvc
+ms.date: 09/29/2021
+no-loc: [Home, Privacy, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: fundamentals/w3c-logger/index
+---
+
+# W3C Logger in ASP.NET Core
+
+W3CLogger is a middleware that writes log files, in the W3C standard format (described [here](https://www.w3.org/TR/WD-logfile.html)), containing information about HTTP requests and HTTP responses. W3C logger provides logs of:
+
+* HTTP request information
+* Common properties
+* Headers
+* HTTP response information
+* Metadata about the request/response pair (date/time started, time taken)
+
+W3C Logging is valuable in several scenarios to:
```

Comment:
Is this about W3C Logging the concept, or W3CLogger (this middleware)? Sounds like the latter, in which case, I'd change this line.